src/validations.js

```diff
@@ -18,8 +19,9 @@ export function useValidationsStore() {
 export async function loadValidations(keyword) {
   const store = useValidationsStore()
   if (!store.loaded) {
-    const file = await getFilesWithKeyword(keyword)
-    const res = await fetch('../../../' + file[0].vUrl)
+    const files = await getFilesWithKeyword(keyword)
+    const path = getPitcherFolderPath() + files[0].vUrl
```

Comment:
would create a function that takes a vURL and returns a path instead of this

Reply:
Ok, like `getFullUrl(vUrl)` ?

Reply:
yeah or getFullPath or something

Reply:
Fixed with 3eaab2b and 1d2d062